### ISSUE TITLE
Thread.isAlive was renamed to Thread.is_alive in python 3.9

### ISSUE THREAD

**Issue by PiRK** [TinosNitso/AutoCove-Plugin#1]
This causes an error when a transaction is received or sent with the development version (run from source) of Electron Cash or Electrum ABC (released versions currently ship with python 3.8, so for now it should still work).

```
Traceback
Traceback (most recent call last):
  File "/home/totor/.electrum-abc/external_plugins/AutoCove-Plugin.zip/AutoCove/qt.py", line 160, in history_updated
    if self.Thread.isAlive(): return    #Don't double broadcast the same return.
AttributeError: 'Thread' object has no attribute 'isAlive'
 ```

**Comment by TinosNitso** [comment]
Thanks for letting me know! I'm guessing I should probably use .is_alive() instead, but I'll need to learn how to run the dev version (never done it). Their might be a new attribute. I'll try figure it out tomorrow. For now I'm spelling out all the OpCodes in **mixed** case, so that it auto-decodes either using OpCodes or OP_CODES (another QCheckBox toggle). e.g. OP_OUTPOINTTXHASH vs OutpointTXHash.

**Comment by PiRK** [comment]
Luckily, running from sources is relatively easy if you are familiar with terminals.

Installing the mandatory dependencies is  basically these 3 commands:
```
git clone https://github.com/Electron-Cash/Electron-Cash; cd Electron-Cash
pip install -r contrib/requirements/requirements.txt --user
pip install pyqt5
```

And then, run with `./electron-cash`, or `python3 ./electron-cash`

Electron Cash's README has additional instructions, if you want the optional dependencies:  https://github.com/Electron-Cash/Electron-Cash/blob/master/README.rst

Thanks for the useful plugin!

**Comment by TinosNitso** [comment]
I've released [v1.0.4](https://github.com/TinosNitso/AutoCove-Plugin/releases/tag/v1.0.4) which should resolve this issue. However I should still check for myself. Thanks again!
